packages/mobile/src/navigator/DrawerTopBar.tsx

```diff
@@ -29,14 +31,15 @@ function DrawerTopBar({ middleElement, rightElement, scrollPosition, testID }: P
     [scrollPosition]
   )
 
+  const hamburgerOnPress = () => {
```

Comment:
nit: we use this convention for naming event handlers.
```suggestion
  const onPressHamburger = () => {
```